src/commands/man.rs

```diff
@@ -0,0 +1,68 @@
+use anyhow::bail;
+use anyhow::Result;
+use reqwest::header;
+
+use crate::serenity;
+use crate::types::Context;
+
+const USER_AGENT: &str = "kangalioo/rustbot";
+
+#[poise::command(
+	prefix_command,
+	slash_command,
+	broadcast_typing,
+	category = "Utilities"
+)]
+pub async fn man(
+	ctx: Context<'_>,
+	#[description = "Section of the man page"] section: Option<String>,
```

Comment:
why didnt you take a number? doesnt poise support that?

Reply:
> why didnt you take a number? doesnt poise support that?

Sorry! I didn't bother checking weather Poise supported numbers. I'll make sure to make the changes in a separate PR.

Reply:
eh dont bother